xds/src/main/java/io/grpc/xds/client/ControlPlaneClient.java

```diff
@@ -436,9 +486,11 @@ private void close(Exception error) {
     }
 
     private void cleanUp() {
+      sentInitialRequest = false;
```

Comment:
Delete. Setting adsStream to null (below) is all that's needed to change the result of isConnected().